docs/File-Formats.md

```diff
@@ -802,21 +801,40 @@ case_list_description: Custom subset of samples (825 samples)
-
-* **All Samples**: "_all" (e.g. "brca_tcga_pub_all").
+#### Case list stable id suffixes
+In order for sample counts to propagate to the data sets widget on the home page and the table on the [Data Sets](http://www.cbioportal.org/public-portal/data_sets.jsp) page, the following case list suffixes need to be used in the stable_id property (e.g. `brca_tcga_pub_sequenced`). This is also needed for correct statistics in the Study view page when calculating the frequency of CNA and of mutations per gene in the respective summary tables.
+
+* **Sequenced**: `_sequenced` . When only a mutation profile is selected on the query page, this is the default case list.
+* **CNA**: `_acgh`. When only a CNA profile is selected on the query page, this is the default case list.
+* **CNA**: `_cna`.
+* **Sequenced and CNA**: `_cnaseq`. When a mutation and CNA genetic profile are selected on the query page, this is the default case list.
+* **mRNA (microarray)**: `_mrna`. When only a mRNA (microarray) profile is selected on the query page, this is the default case list.
+* **mRNA (RNA-Seq)**: `_rna_seq_mrna`. When only a mRNA (RNA-Seq) profile is selected on the query page, this is the default case list.
+* **mRNA (RNA-SeqV2)**: `_rna_seq_v2_mrna`. When only a mRNA (RNA-SeqV2) profile is selected on the query page, this is the default case list.
+* **mRNA normal**: `_normal_mrna`. Used for the datasets page to calculate the number of normal samples.
+* **mRNA normal**: `_microrna`. Used for the datasets page to calculate the number of microRNA samples.
+* **Methylation (HM27)**: `_methylation_hm27`.
+* **RPPA**: `_rppa`. When only a RPPA profile is selected on the query page, this is the default case list.
+* **Sequenced, CNA and mRNA**: `_3way_complete` When a mutation, CNA and mRNA profile are selected on the query page, this is the default case list.
+* **All**: `_all`. If you are not using *add_global_case_list* attribute in [Study metadata](#cancer-study), you need to add this case list.
+
+#### Study view table calculations based on case lists
+This `_sequenced` and `_cna` lists are used to determine the percentage of samples with mutations and CNA in tables on the Study View. If these case list are not given, the system will assume that *all* samples have been sequenced / CNA measured and will calculate the frequency accordingly.
+
+#### Case list categories
+These are the valid case lists categories for `case_list_category: ` in the meta file.
+- `all_cases_in_study`
+- `all_cases_with_mutation_data`
+- `all_cases_with_cna_data`
+- `all_cases_with_log2_cna_data`
+- `all_cases_with_methylation_data`
+- `all_cases_with_mrna_array_data`
+- `all_cases_with_mrna_rnaseq_data`
+- `all_cases_with_rppa_data`
+- `all_cases_with_microrna_data`
+- `all_cases_with_mutation_and_cna_data`
+- `all_cases_with_mutation_and_cna_and_mrna_data`
+- `all_cases_with_gsva_data`
```

Comment:
what about `other` ?

Reply:
I forgot it, but I added it in new commit.